memote/suite/tests/test_consistency.py

```diff
@@ -63,7 +63,7 @@ def test_stoichiometric_consistency(read_only_model):
 @pytest.mark.parametrize("met", [x for x in consistency.ENERGY_COUPLES])
 @annotate(title="Erroneous Energy-generating Cycles", format_type="count",
           data=dict(), message=dict())
-def test_detect_energy_generating_cycles(read_only_model, met):
+def test_detect_energy_generating_cycles(model, met):
```

Comment:
Isn't a `model` a copy and thus more expensive than a `read_only_model`? Wouldn't that slow the tests down?